.cspell.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "version": "0.2",
+  "ignorePaths": ["**/*.json", "**/*.css", "node_modules", "**/*.log"],
+  "useGitignore": true,
+  "language": "en",
+  "words": ["dataurl", "devpool", "outdir", "servedir"],
+  "dictionaries": ["typescript", "node", "software-terms"],
+  "import": ["@cspell/dict-typescript/cspell-ext.json", "@cspell/dict-node/cspell-ext.json", "@cspell/dict-software-terms"],
+  "ignoreRegExpList": ["[0-9a-fA-F]{6}"],
+  "ignoreWords": ["ubiquibot", "issu", "Supabase", "supabase", "SUPABASE"]
```

Comment:
Why is `issu` needed?

Reply:
![image](https://github.com/user-attachments/assets/b213fd68-831e-4bea-88ea-ad25d4981caf)


```
 created%22,%22issu](https:
```

Reply:
Ah I see. Maybe you can fix the typo instead?

Reply:
cspell has weird bugs, I'm assuming there is no typo and its a false positive.